doc/reference-py-sack.rst

```diff
@@ -74,8 +74,7 @@
 
   .. method:: add_cmdline_package(filename)
 
-    Add a package to a command line repository and return it. The package is
-    specified as a string `filename` of an RPM file
+    Add a package to a command line repository and return it. Command line repository will be automatically created if doesn't exist already. It could be referenced later by `@commandline` name. The package is specified as a string `filename` of an RPM file.
```

Comment:
I'd move the new sentences to the end of the old paragraph because I think that the type of the argument is more important than the fact about the repository. I also think that there should be "*The* command line repository will...".